ouroboros-network/src/Ouroboros/Network/PeerSelection/PeerStateActions.hs

```diff
@@ -393,6 +393,11 @@ awaitAllResults tok bundle = do
 -- Internals: peer state & connection handle
 --
 
+-- GR-FIXME[R]: 'PeerState' is never read except through 'getCurrentState', so 
+-- the point of using this (rather than PeerStatus) is what: ?
+--   A. future use?
+--   B. used for tracing? (is it?)
+--   C. am I missing something?
```

Comment:
`updateUnlessCold` updates the value unless the peer is in `cold` state, which rules out `PromotingToWarm`, the other states are indeed not used.